types/validator.go

```diff
@@ -171,3 +172,17 @@ func ValidatorFromProto(vp *tmproto.Validator) (*Validator, error) {
 
 	return v, nil
 }
+
+// DeterministicValidator returns a deterministic validator, useful for testing.
+// UNSTABLE
+func DeterministicValidator(key crypto.PrivKey) (*Validator, PrivValidator) {
```

Comment:
Does this need to live in this file? Do we have a testing utilities file we can put this in instead?

Reply:
I will make it private in the evidence file